locals.tf

```diff
@@ -0,0 +1,29 @@
+locals {
+
+  # Logic to use AD name provided by user input on ORM or to lookup for the AD name when running from CLI
+  availability_domain = (var.availability_domain_name != "" ? var.availability_domain_name : data.oci_identity_availability_domain.ad.name)
+
+  # local.use_existing_network referenced in network.tf
+  use_existing_network = var.network_strategy == var.network_strategy_enum["USE_EXISTING_VCN_SUBNET"] ? true : false
+
+  # local.is_public_subnet referenced in compute.tf
+  is_public_subnet = var.subnet_type == var.subnet_type_enum["PUBLIC_SUBNET"] ? true : false
+
+  # Logic to select Oracle Autonomous Linux 7 platform image (latest image available)
```

Comment:
This actually not latest per se, it's pegged on "Oracle-Autonomous-Linux-7.8-2020.04-0". Tiny tiny issue.